src/Lykke.Sdk/LykkeAppOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using JetBrains.Annotations;
+
+namespace Lykke.Sdk
+{
+    [PublicAPI]
+    public class LykkeAppOptions
+    {
+        public string AppName { get; set; } = System.Reflection.Assembly.GetEntryAssembly().GetName().Name;
```

Comment:
Why not using Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationName?